### PR TITLE
add all the project models

### PR DESCRIPTION
**Pull request by DohaData**
Add the following models:
- User
- Product
- Cart
- Order
- Address

### CODE REVIEW

File: models/Address.model.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            type: Number,
+            required: true,
+        },
+        additionalInfo: {
+            type: String,
+        },
+    },
+    {
+        timestamps: true,
+    });
```
Comment (TunaBonaudi) [inline models/Address.model.js]: YOU FORGOT TO ADD LAST EMPTY LINE

Comment (TunaBonaudi) [reply models/Address.model.js]: ALSO YOU DIDNT EXPORT THE MODEL

**Review by TunaBonaudi — APPROVED** [review]
GOOD JOB

File: models/Address.model.js (hunk 1 above)
Comment (DohaData) [reply models/Address.model.js]: nice catch, thank you ;)

**Pull request merged**